--- spell_rev/components/main_component.tpa	2016-12-26 14:36:26.000000000 +0100
+++ Fixpack/spell_rev/components/main_component.tpa	2017-03-11 16:09:09.239205700 +0100
@@ -1279,8 +1279,11 @@
   SAY NAME2 @437    SAY IDENTIFIED_DESC   @438
 COPY ~spell_rev\spwi1##\spwi120d.spl~     ~override~
 
-COPY ~spell_rev\spwi1##\spwi123.spl~     ~override~  // Find Familiar
-  SAY NAME1 @439    SAY UNIDENTIFIED_DESC @440
+ACTION_IF NOT GAME_IS ~eet~ BEGIN
+  COPY ~spell_rev\spwi1##\spwi123.spl~     ~override~  // Find Familiar
+    SAY NAME1 @439    SAY UNIDENTIFIED_DESC @440
+END
+
 COPY ~spell_rev\spwi1##\scrl6d.itm~     ~override~
   SAY NAME2 @439    SAY IDENTIFIED_DESC   @440
 COPY ~spell_rev\spwi1##\famcat.cre~     ~override~
